homeassistant/components/netatmo/light.py

```diff
@@ -0,0 +1,116 @@
+"""Support for the Netatmo camera lights."""
+import logging
+
+import pyatmo
+
+from homeassistant.components.light import Light
+
+from .camera import CameraData
+from .const import AUTH, DOMAIN, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Netatmo camera light platform."""
+    auth = hass.data[DOMAIN][entry.entry_id][AUTH]
+
+    def get_entities():
+        """Retrieve Netatmo entities."""
+        entities = []
+
+        try:
+            camera_data = CameraData(hass, auth)
+
+            for camera in camera_data.get_all_cameras():
+                if camera["type"] == "NOC":
+                    _LOGGER.debug("Setting up camera %s", camera["id"])
+                    entities.append(NetatmoLight(camera_data, camera["id"]))
+        except pyatmo.NoDevice:
+            _LOGGER.debug("No cameras found")
+
+        return entities
+
+    async_add_entities(await hass.async_add_executor_job(get_entities), True)
+
+
+class NetatmoLight(Light):
+    """Representation of a Netatmo Presence camera light."""
+
+    def __init__(self, camera_data: CameraData, camera_id: str):
+        """Initialize a Netatmo Presence camera light."""
+        self._camera_id = camera_id
+        self._data = camera_data
+        self._camera_type = self._data.camera_data.get_camera(camera_id).get("type")
+        self._name = (
+            f"{MANUFACTURER} {self._data.camera_data.get_camera(camera_id).get('name')}"
+        )
+        self._is_on = False
+        self._unique_id = f"{self._camera_id}-{self._camera_type}-light"
```

Comment:
Do you need `camera_type` if you already have `camera_id` ?

Reply:
Not necessarily. But does it hurt?

Reply:
We need to use the least amount of variables to create a unique value. That way if they change the spelling of a model name etc it won't impact us.